### ISSUE TITLE
Edit channel > Displaying an empty screen after save the changes

### ISSUE THREAD

**Issue by tlle14** [linagora/Twake-Mobile#330]
# Actual Result
After clicking "Save" button to save the changes of a channel name, it redirects to the empty screen. 
It happens sometimes.

**1. When renaming a channel:**
https://user-images.githubusercontent.com/69950655/110443391-70ebb780-80ee-11eb-8208-f1f6cb49f221.mp4

**2. When click save button multiple times**
https://user-images.githubusercontent.com/69950655/110444941-2ec37580-80f0-11eb-8f7a-abb9b4b6d397.mp4

# Expected result
After saving the change, it should display the channel that has been updated.
The app should avoid displaying an empty screen

# Environment
Android App
Build version: 2.2.9-1

**Comment by tlle14** [comment]
Hi @RockinPaul 
Retest failed!
Could you please check? 
https://images.zenhubusercontent.com/309637937/c9f54098-e4f4-4021-980d-d213764fa4db/video9d8f3d27_7189_42c5_a46f_987972eb24d0video.mp4

**Comment by tlle14** [comment]
Retest is blocked by issue https://github.com/linagora/Twake-Mobile/issues/386

**Comment by tlle14** [comment]
Thank @RockinPaul 
Retest pass!
By the way, we have a small issue, I will create another ticket for easier to track. 

https://images.zenhubusercontent.com/309637937/8ec7fba8-d01a-46ac-a2d7-5c03594f5a43/video17774bd6_8b5e_4d32_b9ea_b8aeda609354video.mp4
